client/src/actions/candidateActions.js

```diff
@@ -15,11 +15,40 @@ const fetchCandidatesFailure = (error) => (
   }
 );
 
+const sendAvailabilitySuccess = {
```

Comment:
Can you move these to userActions? This file will be deprecated.

Reply:
didn't we agree that a candidate isn't really a user? I think its worth making a distinction between the actions that an authorised user (ie `userActions`)could take versus an unauthorised candidate

Reply:
They are stored in different tables, but they essentially do the same thing and are both in directory page, which is why they are grouped together. What unauthorized candidates are you thinking of? And if they have different action types, that can be handled in one reducer anyway.

Reply:
I mean unauthorised, as in they haven't had to log in whereas employees have. If you think its better to group them into one reducer though I'm happy to do that; I don't really have a clue on good redux design as I've never used it before this.

Reply:
That is already handled in front end, no need to worry about it :) Yeah it is better to group them together for code reusability because we might be adding admin and/or changing interviewer to admin, it will make things a lot easier for us if we decide to go forward with that. If you don't understand what I'm doing in userActions and directoryReducer, don't hesitate to ask - I'm here to help!

Reply:
Also, do you want all actions in `userActions`, so everything currently in both candidate and interviewer actions?

Reply:
Btw I made a mistake, this should be in `directoryActions` and the corresponding reducer file is `directoryReducer`.

You can see what `getUsers` and `addUser` are doing - it covers all user types.